content/applications/hr/payroll.rst

```diff
@@ -357,27 +550,68 @@ to the default rules, or the creation of new rules, should only be done when nec
-  Code`. Depending on what is selected, the fixed amount, percentage, or Python code needs to be
-  entered next.
+To make a new rule, click :guilabel:`New`. A new rule form appears. Enter the following information
+in the fields.
+
+Top section
+~~~~~~~~~~~
+
+- :guilabel:`Rule Name`: enter a name for the rule. This field is required.
+- :guilabel:`Category`: select a category the rule applies to from the drop-down menu, or enter a
+  new one. This field is required.
+- :guilabel:`Code`: enter a code to be used for this new rule. It is recommended to coordinate with
+  the accounting department for a code to use as this affects accounting reports and payroll
+  processing. This field is required.
+- :guilabel:`Sequence`: enter a number indicating when this rule is calculated in the sequence of
+  all other rules.
+- :guilabel:`Salary Structure`: select a salary structure the rule applies to from the drop-down
+  menu, or enter a new one. This field is required.
+- :guilabel:`Active`: enable this toggle so the rule is available for use. Disable the toggle to
+  hide the rule without deleting in the database.
+- :guilabel:`Appears on payslip`: tick the checkbox to have the rule appear on employee payslips.
+- :guilabel:`View on Employer Cost Dashboard`: tick the checkbox to have the rule appear on the
+  :guilabel:`Employer Cost` report, located on the *Payroll* app dashboard.
+- :guilabel:`View on Payroll Reporting`: tick the checkbox to have the rule appear on payroll
+  reports.
+
+General tab
+~~~~~~~~~~~
+
+Conditions
+**********
+
+- :guilabel:`Condition Based on`: select from the drop-down menu whether the rule is
+  :guilabel:`Always True` (always applies), a :guilabel:`Range` (applies to a specific range, which
+  is entered beneath the selection), or a :guilabel:`Python Expression` (the code is entered beneath
+  the selection). This field is required.
+
+Computation
+***********
+
+- :guilabel:`Amount Type`: select from the drop-down menu whether the amount is a :guilabel:`Fixed
+  Amount`, a :guilabel:`Percentage (%)`, or a :guilabel:`Python Code`. Depending on what is
+  selected, the fixed amount, percentage, or Python code needs to be entered next. This field is
+  required.
+
+Company contribution
+********************
+
+- :guilabel:`Partner`: if another company financially contributes to this rule, select the company
+  from the drop-down menu.
+
+Description tab
+~~~~~~~~~~~~~~~
+
+Provide any additional information in this tab to help clarify the rule. This tab only appears in
+the rule form.
+
+Accounting tab
+~~~~~~~~~~~~~~
+
+- :guilabel:`Debit Account`: select the debit account from the drop-down menu the rule affects.
+- :guilabel:`Credit Account`: select the credit account from the drop-down menu the rule affects.
+- :guilabel:`Not computed in net accountability`: tick the checkbox to delete the value of this rule
+  in the `Net Salary` rule.
 
 .. image:: payroll/new-rule.png
    :align: center
```

Comment:
I think this screenshot would be more useful between 'Top Section' and General Tab...